src/js/tabs/configuration.js

```diff
@@ -701,7 +701,11 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         if (semver.gte(CONFIG.apiVersion, "1.42.0"))  {
             serialRXtypes.push('DJI_HDL_7MS');
         }
-
+        
+        if (semver.gte(CONFIG.apiVersion, "1.42.0"))  {
+            serialRXtypes.push('SPEKTRUM SRXL2');
```

Comment:
Since `DJI_HDL_7MS` and `SPEKTRUM SRXL2` are both added in the same version you should add your push to that block, right after the DJI one.

Reply:
Ah right. Did a force push with the change amended to last commit with the correction

Reply:
Take a look to the spi push elemnts a little later in the code. We use only one call to the push method passing all the elements needed. For coherence is better to maintain only one push and not one for element.

Reply:
> Take a look to the spi push elemnts a little later in the code. We use only one call to the push method passing all the elements needed. For coherence is better to maintain only one push and not one for element.

I've addressed the issue in the last commit. :)